src/components/ColumnComponent.module.css

```diff
@@ -33,6 +34,6 @@
     font-size: 17px;
 }
 
-.buttonPlus:hover {
-    cursor: pointer;
-}
+.buttonPlus, .buttonText {
+    opacity: 75%;
```

Comment:
The opacity property can take a value from 0.0 - 1.0. Not %
https://www.w3schools.com/css/css_image_transparency.asp